pkg/controllers/dashboard/helm/repo.go

```diff
@@ -247,6 +255,23 @@ func (r *repoHandler) download(repoSpec *catalog.RepoSpec, status catalog.RepoSt
 	return status, nil
 }
 
+func (r *repoHandler) checkConfigMap(repo *catalog.ClusterRepo, status *catalog.RepoStatus) error {
+	if repo.Spec.GitRepo != "" && status.IndexConfigMapName != "" {
+		// check if the IndexConfigMap exists
```

Comment:
This comment is redundant as code below obviously does what the comment says.

Reply:
the comment is removed